controller/internal/enforcer/utils/nsenter/nsenter.c

```diff
@@ -57,4 +245,19 @@ void nsexec(void) {
     setenv("APORET_ENV_NSENTER_ERROR_STATE",strerror(errno),1);
   }
   
+  
+}
+
+
+
+
+void setupiptables() {
+  char *container_pid_env = getenv("TRIREME_ENV_CONTAINER_PID");
+  if (container_pid_env == NULL){
+    chmod("/run/xtables.lock",0666);
+  }
+  return;
```

Comment:
That is too drastic. We are opening a hole for everyone. We need to think of alternatives.